src/main/java/model/line/RandomLineGenerator.java

```diff
@@ -0,0 +1,25 @@
+package model.line;
+
+import static java.util.stream.Collectors.collectingAndThen;
+import static java.util.stream.Collectors.toList;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.stream.IntStream;
+import model.bridge.Bridge;
+
+public class RandomLineGenerator implements LineGenerator {
+    @Override
+    public Line generateLine(int width) {
+        return IntStream.range(0, width)
+                .mapToObj(i -> getRandomBridge())
+                .collect(collectingAndThen(toList(), Line::new));
+    }
+
+    private static Bridge getRandomBridge() {
```

Comment:
```suggestion
    private Bridge getRandomBridge() {
```
메서드 분리할 때 static 이 포함되죠? 😂 
이거 method 분리 시도했을 때 옆에 설정 버튼 누르고 체크박스 제거하면 default 로 static 안 붙게 할 수 있어요 :
<img width="487" alt="Screenshot 2024-03-01 at 5 12 09 PM" src="https://github.com/woowacourse/java-ladder/assets/48251668/e42ed381-b05c-4c66-97ee-b05cec6649e1">

Reply:
팁 감사합니다!!👍